src/components/Checkout/OrderTotals.vue

```diff
@@ -391,11 +392,11 @@ export default {
 				price: numeral(this.totals.donationTotal)
 			};
 		},
-		isCorporateCampaignPage() {
-			return this.$route.path.substring(0, 4) === '/cc/';
-		}
```

Comment:
I think you should keep this computed prop in place and replace it's contents with `return isCCPage(this.route);`. You would also leave in place all instances of this computed properties name.

The way you have this up now, makes it so the `ccPageCheck` + nested `isCCPage` function get called many times instead of using a computed property which will caluculate 1 time and be reused.

The pattern I'm describing here is also applicable to inContextCheckout and BasketItem components too.